### DISCUSSION TITLE
Receive payments to customer and apply to future invoice

### DISCUSSION THREAD

**Discussion by toddlunn** [cytech/BillingTrack-Wired#12] [Q&A]
Great job on creating this billing system

Is it possible to receive deposits or apply a payment to the customer and then pay out the invoice with the overpayment at a later time.

**Comment by cytech** [comment]
currently no. An invoice must exist.
you could create a $0 invoice.
then in system-settings -> invoices, turn on Allow Entering Payments on Invoices Without Balance:. and enter a payment against the invoice.
balance will go negative.
this can get confusing though

**Comment by toddlunn** [comment]
I did try but the $0 balance invoice did not show in the payment window.
 Do you offer customization?

**Comment by cytech** [comment]
not at this time
i need to create a credit memo function to handle this correctly.
but i do not see this happening in the near future
